tests/roles/glance_adoption/tasks/main.yaml

```diff
@@ -1,9 +1,23 @@
 - name: deploy podified Glance with local backend
-  ansible.builtin.shell: |
-    {{ shell_header }}
-    {{ oc_header }}
-    oc patch openstackcontrolplane openstack --type=merge --patch-file={{ role_path }}/files/glance_local.yaml
-  when: glance_backend == 'local'
+  when: glance_backend == 'nfs'
```

Comment:
`glance_backend` where this variable is defined?

Because glance doesn't have any backend named nfs.

Reply:
This is defined in the job and passed to the playbook to identify the use case we're going to cover (it's not something that goes to enabled_backends, where we have File)